core/src/util/mod.rs

```diff
@@ -107,39 +110,53 @@ mod filtering {
     }
 
     #[test]
-    fn test_is_positive_match() {
-        assert!(is_match("filter", &to_name_regex("%")));
-        assert!(is_match("filter", &to_name_regex("filter")));
-        assert!(is_match("downtimes", &to_name_regex("downtimes")));
-        assert!(is_match("customerssales", &to_name_regex("customer_sales")));
-        assert!(is_match("myiphone", &to_name_regex("my_phone")));
-        assert!(is_match("conversions2022", &to_name_regex("conversions%")));
-        assert!(is_match("integration_test", &to_name_regex("%test")));
-        assert!(is_match("money$$bags", &to_name_regex("money$$bags")));
-        assert!(is_match("money$.bags", &to_name_regex("money$.bags")));
+    fn test_is_positive_match_odbc_2() {
```

Comment:
[super nit] What about updating the test names too and using test_is_positive_match_literal, test_is_negative_match_literal, test_is_positive_match_pattern test_is_negative_match_pattern?